src/AirshipBindings.iOS.Automation/build.gradle

```diff
@@ -28,7 +28,7 @@ task generateBindings {
 
```

Comment:
Is this task still valid? we generate bindings with Xamboni now

Reply:
It is not valid. I could remove.